pkg/util/k8s/k8s.go

```diff
@@ -1385,6 +1385,81 @@ func DeletePrometheus(
 	return k8sClient.Update(context.TODO(), prometheus)
 }
 
+// CreateOrUpdateAlertManager creates a AlertManager object if not present, else updates it
+func CreateOrUpdateAlertManager(
+	k8sClient client.Client,
+	alertManager *monitoringv1.Alertmanager,
+	ownerRef *metav1.OwnerReference,
+) error {
+	existingAlertManager := &monitoringv1.Alertmanager{}
+	err := k8sClient.Get(
+		context.TODO(),
+		types.NamespacedName{
+			Name:      alertManager.Name,
+			Namespace: alertManager.Namespace,
+		},
+		existingAlertManager,
+	)
+	if errors.IsNotFound(err) {
+		logrus.Debugf("Creating AlertManager %s/%s", alertManager.Namespace, alertManager.Name)
```

Comment:
Can we change to "AlertManager does not exist, creating a new AlertManager"

Reply:
We have a separate message when we update the alert manager object on line 1420. When we say `creating`, it is only when the object does not exist.
I don't mind making it explicit, but all other k8s util functions just print "Creating/Updating/Deleting <kind> <namespace>/<name>". We should change it everywhere for cleaner logging.